### DISCUSSION TITLE
Hi i am getting this error can and i have cudan an dnn installed.

### DISCUSSION THREAD

**Discussion by frankool** [vietanhdev/anylabeling#63] [Q&A]
[ WARN:29@235.138] global net_impl.cpp:174 cv::dnn::dnn4_v20221220::Net::Impl::setUpNet DNN module was not built with CUDA backend; switching to CPU

**Comment by vietanhdev** [comment]
Hi @frankool !
- Which model are you using? (SAM/YOLO)?
- Can you try to build and install opencv-python with CUDA? [https://gist.github.com/raulqf/f42c718a658cddc16f9df07ecc627be7](https://gist.github.com/raulqf/f42c718a658cddc16f9df07ecc627be7)
Thank you!
